can/interfaces/vector/canlib.py

```diff
@@ -49,9 +52,50 @@
     LOG.warning("Could not import vxlapi: %s", exc)
 
 
+def deprecated_args_alias(**aliases):
+    def deco(f):
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            rename_kwargs(f.__name__, kwargs, aliases)
+            return f(*args, **kwargs)
+
+        return wrapper
+
+    return deco
+
+
+def rename_kwargs(func_name, kwargs, aliases):
+    for alias, new in aliases.items():
+        if alias in kwargs:
+            warnings.warn(
+                "{} is deprecated; use {}".format(alias, new), DeprecationWarning
+            )
+            if new in kwargs:
+                raise TypeError(
+                    "{} received both {} (deprecated) and {}".format(
+                        func_name, alias, new
+                    )
+                )
+            kwargs[new] = kwargs.pop(alias)
+
+
+def arg_to_c_uint(value):
+    return ctypes.c_uint(int(value))
```

Comment:
I'd prefix this function with an underscore to mark it internal.

A better place for it would be `can.ctypesutil`